### DISCUSSION TITLE
Fine-tuning the Model with Korean Dataset

### DISCUSSION THREAD

**Discussion by leeseungchae** [MahmoudAshraf97/ctc-forced-aligner#44] [Q&A]
The model is outstanding, and I’d like to train it further with a Korean dataset to improve performance for my specific use case. Could you recommend any repositories for reference or briefly explain how to fine-tune the model with additional data?

Any guidance or resources would be greatly appreciated!

**Comment by MahmoudAshraf97** [comment]
Check [This](https://huggingface.co/MahmoudAshraf/mms-300m-1130-forced-aligner/discussions/1)
